backend/ddang/src/main/java/com/ddang/ddang/chat/infrastructure/persistence/QuerydslChatRoomRepositoryImpl.java

```diff
@@ -20,43 +16,29 @@ public class QuerydslChatRoomRepositoryImpl implements QuerydslChatRoomRepositor
 
     private final JPAQueryFactory queryFactory;
 
-    @Override
-    public List<ChatRoom> findAllByUserId(final Long userId) {
-        return findChatRoomJPAQuery().where(isSellerOrWinner(userId))
-                                     .orderBy(chatRoom.id.desc())
-                                     .fetch();
-    }
-
-    private JPAQuery<ChatRoom> findChatRoomJPAQuery() {
-        return queryFactory.selectFrom(chatRoom)
-                           .leftJoin(chatRoom.buyer).fetchJoin()
-                           .leftJoin(chatRoom.auction, auction).fetchJoin()
-                           .leftJoin(auction.subCategory, category).fetchJoin()
-                           .leftJoin(category.mainCategory).fetchJoin()
-                           .leftJoin(auction.seller).fetchJoin()
-                           .leftJoin(auction.auctionImages).fetchJoin();
-    }
-
-    private BooleanExpression isSellerOrWinner(final Long userId) {
-        return (auction.seller.id.eq(userId))
-                .or(chatRoom.buyer.id.eq(userId));
-    }
-
     @Override
     public Optional<ChatRoom> findChatRoomById(final Long chatRoomId) {
-        final ChatRoom findChatRoom = findChatRoomJPAQuery()
-                .where(chatRoom.id.eq(chatRoomId))
-                .fetchOne();
+        final ChatRoom findChatRoom = queryFactory.selectFrom(chatRoom)
+                                                  .leftJoin(chatRoom.buyer).fetchJoin()
+                                                  .leftJoin(chatRoom.auction, auction).fetchJoin()
+                                                  .leftJoin(auction.seller).fetchJoin()
+                                                  .leftJoin(auction.auctionImages).fetchJoin()
+                                                  .leftJoin(auction.lastBid).fetchJoin()
+                                                  .where(chatRoom.id.eq(chatRoomId))
+                                                  .fetchFirst();
 
         return Optional.ofNullable(findChatRoom);
     }
 
     @Override
-    public Optional<ChatRoom> findByAuctionId(final Long auctionId) {
-        final ChatRoom findChatRoom = findChatRoomJPAQuery()
-                .where(auction.id.eq(auctionId))
-                .fetchOne();
-
-        return Optional.ofNullable(findChatRoom);
+    public Optional<Long> findChatRoomIdByAuctionId(final Long auctionId) {
+        final ChatRoom findChatRoom = queryFactory.selectFrom(chatRoom)
+                                                  .where(chatRoom.auction.id.eq(auctionId))
+                                                  .fetchFirst();
```

Comment:
```suggestion
        final Long findChatRoomId = queryFactory.select(chatRoom.id)
                                                  .from(chatRoom)
                                                  .where(chatRoom.auction.id.eq(auctionId))
                                                  .fetchFirst();
```

id만 찾는 것 같은데 굳이 ChatRoom 전체를 조회할 필요는 없을 것 같습니다